test/AutoDelegateBravoGovernor.t.sol

```diff
@@ -4,14 +4,28 @@ pragma solidity ^0.8.23;
 import {Test} from "forge-std/Test.sol";
```

Comment:
*arbitrary comment location*

Across the board, the test naming here needs to be updated to match our `scopelint spec` requirements, so that the output will read as a test-driven spec.

Reply:
Done: [a37191e](https://github.com/withtally/stGOV/pull/87/commits/a37191e485c43446f2517ce634a60d551e69ddec)